saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRLConfigurator.scala

```diff
@@ -13,4 +15,5 @@ object mSpRLConfigurator {
   val expName = "BM" //{"BM", "BM+C", "BM+C+E", "BM+C+E+I"}
   val isTrain = true
   val iterations = 50
+  val useConstraints = false
```

Comment:
In the SRL app, you have gone the other way, defined the parameters in the configurator and the expName in the app. do you want to change this to follow that approach?